lib/statsd/instrument.rb

```diff
@@ -58,17 +58,13 @@ def self.generate_metric_name(metric_name, callee, *args)
 
     if Process.respond_to?(:clock_gettime)
       # @private
-      def self.duration
-        start = Process.clock_gettime(Process::CLOCK_MONOTONIC)
-        yield
-        Process.clock_gettime(Process::CLOCK_MONOTONIC) - start
+      def self.gettime
```

Comment:
This method is part of the public API. We cannot really change it like this. I know we have code out there that uses this function